feature/dashboard/src/main/res/layout/fragment_main.xml

```diff
@@ -60,4 +60,44 @@
 
         </LinearLayout>
     </ScrollView>
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:id="@+id/rootedDeviceError"
+        android:visibility="gone"
+        android:background="@color/simprints_red">
+        <TextView
+            android:id="@+id/alertTitle"
```

Comment:
If the TextView and ImageView will have IDs better to have clearer ones.

Reply:
@BurningAXE Ok I will remove the IDs, I copied them from the alert screen :D

Reply:
Can you use the alert screen for this? Similar to how it is handled in the regular flows (in the login-check module)

Reply:
@luhmirin-s We only show the alert screen in the workflows, not in the dashboard
I tried to use it and found it complex to import as the alert screen stores alert events, which is not supported outside the intent workflows.